woocommerce/woocommerce
# Store Customization > Update the position for patterns on templates two and three

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR updates the order of the patterns for templates 2 and 3 in the Customize Your Store (CYS) flow, as requested by design.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

Make sure you are testing this with a Jetpack connected instance of WooCommerce, as the AI text completion endpoint can only be called with a Jetpack connected account.

1. Create a new WooCommerce installation with this version.
2. Install and activate the WooCommerce Blocks plugin from this zip file: [woocommerce-gutenberg-products-block.zip](https://github.com/woocommerce/woocommerce/files/12806758/woocommerce-gutenberg-products-block.zip)
3. Make sure to enable customize-store feature flag
4. Go to WooCommerce -> Home and click Customize Store task
5. Click Design with AI button
6. Fill out a business description and proceed
7. From the Assembler sidebar, click on "Change Your Homepage":

<img width="619" alt="Screenshot 2023-10-04 at 20 08 15" src="https://github.com/woocommerce/woocommerce/assets/15730971/cd2cb4c6-11cb-4ab8-833c-ef352ee008a1">

8. Make sure you can visualize all three homepage templates as expected, and the patterns for the second and third templates are as follows:

https://github.com/woocommerce/woocommerce/assets/15730971/58b42374-f3d8-4311-a493-91b95081d187

Note that AI dynamically changes the images and content, so those are not expected to be an exact match when compared with the video. You may notice some oddities with spacing between patterns on Template 3: this will be changed at the pattern level on WooCommerce Blocks. 



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Update the order of the patterns for templates in the Customize Your Store flow

#### Comment 

</details>
